src/components/MiniBlog/index.tsx

```diff
@@ -215,7 +215,17 @@ const MiniBlog: React.FC = () => {
                     image={`/blog/images/${elm.slug}.png`}
                   />
                   <CardContent>
-                    <CustomTag blogLabel={elm.tags} />
+                    <div className={classes.wrapperBlock}>
+                      <CustomTag blogLabel={elm.tags} />
+                      <p className={classes.readTime}>
+                        <img
+                          src="../Images/svg/time-five.svg"
+                          alt={t("header.submitAlt")}
+                          className={classes.rightSpacing}
+                        />
+                       {`${readingTime(elm.content)} min read`}
```

Comment:
```suggestion
                       {`${readingTime(elm.content)} mins read`}
```

Reply:
we don't call `mins` while displaying the estimated read time, it's always `min` read.